src/Loader.php

```diff
@@ -310,6 +315,14 @@ public static function register_scripts() {
 
 		wp_set_script_translations( 'wc-currency', 'woocommerce-admin' );
 
+		wp_register_script(
+			'wc-customer-effort-score',
+			self::get_url( 'customer-effort-score/index', 'js' ),
+			array(),
```

Comment:
Probably yes, this seems to happen work without it so far, but I'll put it in.

Reply:
Actually no! The `customer-effort-score` component is independent of the measurement method - `CustomerEffortScoreTracks` provides a function to `CustomerEffortScore` that uses wc-tracks to actually report the metric. This is so that other teams can use CustomerEffortScore without needing to depend on Tracks.

Reply:
Ah, right - whoops!